internal/dag/builder_test.go

```diff
@@ -3527,6 +3528,16 @@ func TestDAGInsert(t *testing.T) {
 		},
 	}
 
+	cert2 := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "ca",
+			Namespace: "caCertOriginalNs",
```

Comment:
@stevesloka, @youngnick in my unit testing I see that the lookup for this secret in the kubernetes cache fails. Could this be because this namespace is somehow not present in the cache?

Reply:
I think somehow the namesapace/name is getting setup wrong. If you look at where the cert gets looked up is: 

`cacertoriginalns/cacertoriginalns/ca` when it should be `cacertoriginalns/ca`

I'm looking here for this value: https://github.com/projectcontour/contour/blob/7d7ffea619dcb45cb9c7e0406b37047fc9fcae46/internal/dag/cache.go#L565

Reply:
Thanks @stevesloka, fixed it.